docs/_docs/dev-guide/tavern.md

```diff
@@ -89,53 +98,60 @@ ENABLE_TEST_DATA=1 go run ./tavern
 * An authorization handler (`/oauth/authorize`) which users are redirected to by Google after completing Google's OAuth consent flow
-    * This handler is responsible for obtaining a user's profile information from Google using an OAuth access token, and creates the user's account if it does not exist yet
+  * This handler is responsible for obtaining a user's profile information from Google using an OAuth access token, and creates the user's account if it does not exist yet
 
 ##### Trust on First Use
+
 Tavern supports a Trust on First Use (TOFU) authentication model, meaning the first user to successfully authenticate will be granted admin permissions. Subsequent users that login will have accounts created, but will require activation before they can interact with any Tavern APIs. Only admin users may activate other users.
 
-# User Interface
+## User Interface
 
-# CDN API
+## CDN API
+
+### Uploading Files
 
-## Uploading Files
 * File uploads require 2 form parameters: `fileName` and `fileContent`
 * A successful response returns JSON with the following content: `{"data":{"file":{"id":<FILE_ID>}}}`
 
-## Downloading Files
+### Downloading Files
+
 * TODO (CDN is not yet added to Tavern)
 
-# GraphQL API
+## GraphQL API
+
+### Playground
 
-## Playground
 If you'd like to explore the Graph API and try out some queries, head to the `/graphiql` endpoint of your Tavern deployment. This endpoint exposes an interactive playground for you to experiment with GraphQL queries.
 
 ![/assets/img/tavern/graphiql.png](/assets/img/tavern/graphiql.png)
 
+### Creating a New Model
 
-## Creating a New Model
 1. Initialize the schema `cd tavern && go run entgo.io/ent/cmd/ent init <NAME>`
 2. Update the generated file in `tavern/ent/schema/<NAME>.go`
 3. Ensure you include a `func (<NAME>) Annotations() []schema.Annotation` method which returns a `entgql.QueryField()` annotation to tell entgo to generate a GraphQL root query for this model (if you'd like it to be queryable from the root query)
 4. Update `tavern/graphql/gqlgen.yml` to include the ent types in the `autobind:` section (e.g.`- github.com/kcarretto/realm/tavern/ent/<NAME>`)
 5. **Optionally** update the `models:` section of `tavern/graphql/gqlgen.yml` to bind any GraphQL enum types to their respective `entgo` generated types (e.g. `github.com/kcarretto/realm/tavern/ent/<NAME>.<ENUM_FIELD>`)
 6. Run `go generate ./tavern/...` from the project root
 7. If you added an annotation for a root query field (see above), you will notice auto-generated the `query.resolvers.go` file has been updated with new methods to query your model (e.g. `func (r *queryResolver) <NAME>s ...`)
-    * This must be implemented (e.g. `return r.client.<NAME>.Query().All(ctx)` where <NAME> is the name of your model)
+    * This must be implemented (e.g. `return r.client.NAME.Query().All(ctx)` where NAME is the name of your model)
```

Comment:
nit: Breaking consistente naming of <NAME> in docs above and below.

Reply:
Resolved